chpl-src/announcing-chapel-2.6.chpl

```diff
@@ -0,0 +1,463 @@
+  your program. These improvements also lay the groundwork for
+  additional improvements in the future.
+
+  #### Prototype Parallel Debugger
+
+  The Chapel 2.6 release also includes a new tool,
+  `chpl-parallel-dbg`. This tool enables vastly improved debugging of
+  multilocale Chapel programs, which has been a longstanding
+  challenge. It works by launching a debugger on each locale of a
+  Chapel program, and then connecting them all together with a single
+  interface.
+
+  In the following example, we'll demonstrate the use of
+  `chpl-parallel-dbg` on this multilocale Chapel program:
+
+  {{<file_download fname="multiloc/example.chpl" lang="Chapel">}}
+
+  This session shows the program running on two locales, where we
+  switch between them using a custom `on` command that mirrors the
+  Chapel syntax.  To get each locale running, we use a `c` (continue)
+  command, and then are notified once we hit one of the breakpoints.
+  Note that the `up` command is performed automatically by the tooling
+  for us to ensure that we're in the user-level Chapel code.
+
+  {{< figure class="fullwide" src="parallel-dbg.png" >}}
+
+
+  This new tool is still a work in progress, but it's already a huge
+  step forward for debugging multilocale Chapel programs.  To learn
+  more about it or to try it yourself, see [its
+  documentation](https://chapel-lang.org/docs/2.6/usingchapel/debugging/multilocale.html#chpl-parallel-dbg).
+  We are excited to continue improving this tool in future releases.
+
+
+  ### Unit testing Improvements for Mason and VSCode
+
+  This release also includes many significant improvements to `mason`,
+  [Chapel's package
+  manager](https://chapel-lang.org/docs/2.6/tools/mason/mason.html),
+  primarily focused on improving the developer experience when
+  testing.  We fixed many bugs with mason itself and added some new
+  testing features.  For example, suppose we have a simple program
+  defining some unit tests:
+
+  {{<file_download fname="myTest.chpl" lang="Chapel">}}
+
+
+  This can be run, standalone, using:
+
+  ```console
+  $ chpl myTest.chpl
+  $ ./myTest
+  ```
+
+  However, using `mason` we can just run:
+
+  ```console
+   $ mason test myTest.chpl
+  ```
+
+  and it will handle compilation and execution, while also generating
+  a nice report for us:
+
+  ```text
+  
+  ----------------------------------------------------------------------
+  Ran 2 tests in 18.6117 seconds
+  
+  OK (passed = 2 )
+  ```
+
+
+  Chapel 2.6 also adds the ability to selectively run tests by name.
+  For example, this command:
+
+  ```console
+   $ mason test myTest.chpl --filter myTest1
+  ```
+
+  will only run the `myTest1` test.  This is particularly useful when
+  you are working on a specific test and don't want to run the entire
+  suite.
+
+  We also integrated testing into VSCode, so that you can run (and in
+  the future, debug) tests directly from the editor!
+
```

Comment:
the parenthetical could be a side note

Reply:
Engin doesn't like side notes that are too short, and has convinced me he's right, so I left it as you submitted it.  :)